apps/src/templates/page.html.ejs

```diff
@@ -30,6 +30,26 @@
   </div>
 <% }; -%>
 
+
+<% var gameButtons = function() { -%>
+  <div id="gameButtons">
+    <button id="runButton" class="launch blocklyLaunch <%= hideRunButton ? 'invisible' : ''%>">
+      <div><%= msg.runProgram() %></div>
+      <img src="<%= assetUrl('media/1x1.gif') %>" class="run26"/>
+    </button>
+    <button id="resetButton" class="launch blocklyLaunch" style="display: none">
+      <div><%= msg.resetProgram() %></div>
+      <img src="<%= assetUrl('media/1x1.gif') %>" class="reset26"/>
+    </button>
+    <% if (data.controls) { %>
+    <%- data.controls %>
+    <% } %>
+    <% if (!data.pinWorkspaceToBottom && data.extraControlRows) { %>
+    <%- data.extraControlRows %>
+    <% } %>
```

Comment:
Is there not a way to do a block of embedded JS instead of each line having <% %>?